src/Mod/BIM/importers/importDAE.py

```diff
@@ -53,29 +54,34 @@ def translate(context, text):
         return text
     # \endcond
 
-DEBUG = True
+def xml_escape(text: str, entities: dict[str, str] = None) -> str:
+    """Escape text for XML.
+
+    This is a wrapper around xml.sax.saxutils.escape that replaces also
+    `"` with `&quot;` by default.
+    """
+    if entities is None:
+        entities = {'"': "&quot;"}
```

Comment:
If `entities` is *not* `None`, would it make more sense to append the quote substitution, rather than not do it at all? If someone specifically *didn't* want that substitution they could just call `sax_escape` directly.